backend/molgenis-emx2-sql-it/src/test/java/org/molgenis/emx2/sql/TestGrantRolesToUsers.java

```diff
@@ -195,4 +196,19 @@ public void testRole() {
       database.clearActiveUser();
     }
   }
+
+  @Test
+  public void testCaseSensitiveSchemaNames() {
+    // following bug
+    Schema s1 = database.dropCreateSchema("testCaseSensitiveSchemaNames");
```

Comment:
what is dropCreateSchema :S  ? has to read the impl to find out  , suggest recreateSchema

Reply:
would dropAndCreateSchema work for you as well? Would stay closer to psql nomenclature

Reply:
had to look up 'nomenclature' :D , but sure its a bit more clear to me, personally i would prefer having a create and a drop function with a optional dropIfExist param ( or a fail if does not exist  ) but dropAndCreateSchema does indicate what is going on

Reply:
with that param is a more expensive refactor :-). Happy to rename though. Could also make it createOrReplaceIfExists() if you like that better

Reply:
leave it up to you to decide, not a blocking issue , just some thing i had to look up